intl_translation_json/test/multi_json_format_test.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:intl_translation_format/intl_translation_format.dart';
+
+import 'package:intl_translation_format/test_utils.dart';
+import 'package:intl_translation_json/src/multi_json_format.dart';
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
The guide says JSON format is monolingual, but we have both here. Should probably mention that we actually did both in the guide, and maybe have a comment here explaining what it means.